packages/addons/service/net-snmp/source/resources/settings.xml

```diff
@@ -6,7 +6,11 @@
 		<setting label="32002" id="LOCATION" type="text" default="HOME"/>
 		<setting label="32003" id="CONTACT" type="text" default="root@localhost"/>
 		<setting label="32004" id="SNMPVERSION" type="labelenum" values="v1|v2c|v3" default="v2c"/>
-		<setting label="32005" id="SNMPUSER" type="text" default="user"/>
+		<setting label="32007" type="lsep"/>
+		<setting label="32005" id="SNMPUSERI" type="text" default="user"/>
```

Comment:
`SNMPUSERI ` typo (trailing `I`)?

Reply:
Ho yes !! 
sorry doing this late tomorrow night :)
I fix ^^

Reply:
Done !

Thx for the review ^^